data/devices/leeco.yml

```diff
@@ -171,14 +171,12 @@
       - /dev/block/mmcblk0p4
       - misc
  
-  boot_ui:
+ boot_ui:
```

Comment:
The indentation here was correct before (2 spaces). Can you please fix that? It caused the build to fail.

From the build logs:
```
21:57:46 leeco.yml: Failed to convert file: yaml-cpp: error at line 174, column 2: end of sequence not found
```
https://jenkins.noobdev.io/job/DualBootPatcher_Build/639/label_exp=dualbootpatcher-builder,target=android-all-gcc/console